roles/openshift_openstack/tasks/main.yml

```diff
@@ -0,0 +1,20 @@
+- name: Configure Openstack integration
```

Comment:
I'd like to see this role renamed openshift_cloud_provider.

There could be a simple main.yml file that includes the correct template conditionally based on the detected cloud provider.

I'd also like to see a fact from openshift_facts used to do the conditional instead of openshift_use_openstack...  You should be able to use:
```
when: "'provider' in openshift and openshift.provider.name == 'openstack'"
```
I'm not sure if that will cause errors in the master if the attributes from cloud.conf are not valid though.

It would also be nice if the auth-url, username, password, and tennant-id were pulled from commonly used env variables and/or config files.